model/convert/service_event.go

```diff
@@ -943,57 +968,39 @@ func convertClusterQCVotes(cdcClusterQCs []cadence.Value) (
 // convertDKGKeys converts hex-encoded DKG public keys as received by the DKG
 // smart contract into crypto.PublicKey representations suitable for inclusion
 // in the protocol state.
-func convertDKGKeys(cdcDKGKeys []cadence.Value) (
-	groupKey crypto.PublicKey,
-	participantKeys []crypto.PublicKey,
+func convertDKGKeys(cdcDKGKeys ...cadence.Value) (
+	convertedKeys []crypto.PublicKey,
 	err error,
 ) {
 
```

Comment:
previously, we had conceptually different returns (group key; individual key shares). This is no longer the case and therefore I think the risk of confusing the function's return values are negligible now. Hence, we could write the code more compactly:
```suggestion
func convertDKGKeys(cdcDKGKeys ...cadence.Value) ([]crypto.PublicKey, error) {
```